README.md

```diff
@@ -1,2 +1,69 @@
-# SLMM
-The coding exercise for server side applicants
+SLMM - The coding exercise for server side applicants
+=====================================================
+
+Step 1 – all server-side positions
+----------------------------------
+Assume we are interested in creating a smart lawn mowing machine called SLMM (smart lawn mowing machine). Your task is to create the software that will run in the SLMM itself and will be responsible of doing the following actions:
+
+1. Turn the SLMM 90o left or right -> This should take 15 seconds to do
+2. Move forward by one position -> This takes 30 seconds to do
+3. Mow the lawn at the current position -> This takes 120 seconds to do
+
+The application you will create should be able to accept the above commands and do them. Since you do not have access to the actual hardware, for now the only thing required is to accept input from console and output the result (the action being done along with the position of the SLMM) in the console. During application startup, the SLMM is given a set of dimensions (passed in as command line parameters) that represent the width and length of the garden that the SLMM is in.
```

Comment:
I'm not really OK with your version either tbh. I think a better version would be to say "and write the status at the end of the action in the console (write the completed action along with the position of the SLMM)"